### ISSUE TITLE
Does not check nested function calls

### ISSUE THREAD

**Issue by badsyntax** [edvardchen/eslint-plugin-i18next#40]
Hi there. I've noticed this plugin does not check function calls that are more than 1 level deep, for example:

```jsx
import "./styles.css";
import React, { useEffect } from "react";

function t() {
  //
}

function action() {
  //
}

function dispatch() {
  //
}

export default function App() {
  useEffect(() => {
    action("hello"); // shows error
    dispatch(action("hello")); // does not show error
  });
  return <div className="App">{t("foo")}</div>;
}
```

Screenshot:

![Screenshot 2021-03-05 at 06 55 59](https://user-images.githubusercontent.com/102141/110078664-fd5d4980-7d7f-11eb-9b28-c254c0d30e1f.png)

I've attached an archive of a small project (downloaded from codesandbox) that shows the error:

[y9dzx (1).zip](https://github.com/edvardchen/eslint-plugin-i18next/files/6088639/y9dzx.1.zip)

Is there any way I can configure the plugin to check the function call `dispatch(action("hello"));`? In this case I'm using redux dispatch to add an error message to the store and would like to get the action payload validated.

**Comment by edvardchen** [comment]
It has nothing to do with the level. The plugin just ignore the `dispatch` function. See the `Redux/Vuex`  example in README
